custom_components/ha_washdata/__init__.py

```
@@ -239,19 +256,86 @@ async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    except Exception:  # pylint: disable=broad-exception-caught
+        acct = {}
+    if acct:
+        _account_preserved = False
+        try:
+            if acct.get("refresh_token") and not store_account.get_account(hass).get("refresh_token"):
+                await store_account.async_set_account(hass, {
+                    "refresh_token": acct.get("refresh_token"),
+                    "uid": acct.get("uid"), "name": acct.get("name"),
+                })
+            _account_preserved = True
+        except Exception:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)